test/integration/integration_test.go

```diff
@@ -55,6 +55,16 @@ func TestMain(m *testing.M) {
 	var godogPaths = flag.String("paths", "./features", "")
 	flag.Parse()
 
+	if *godogTags != "" {
+		*godogTags += "&&"
+	}
+	runner := util.MinishiftRunner{CommandPath: givenPath}
+	if runner.IsCDK() {
```

Comment:
@hferentschik +1 on `@minishift-only`, going to change it next update.

> This makes CDK visible in our tests. I thought we said, we would keep CDK specific tests in the downstream repo

It is possible I missed this, lets discuss this soon.

> If we have several of these cases, the feature file will become unwieldy

Agree. But hopefully we would not need many of these cases in basic.feature file. I still have the idea of ideally having some cdk.feature or more features tagged whole as `@cdk-only` which would focus on aspects of CDK. And then keep `@cdk-only` on individual scenarios/cases (added up or downstream) to be a rare case.

> It might be confusing for contributors. It is not clear what 'CDK' means from a community perspective.

If documented it could be fine, but still it would be a little bit "strange".